### PR TITLE
add new target FLMO-BANDERAF7.config

### PR DESCRIPTION
**Pull request by cvetaevvitaliy**
add new target FLMO-BANDERAF7.config

### CODE REVIEW

File: configs/default/FLMO-BANDERAF7.config [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM20602
+#define USE_GYRO_SPI_ICM20602
+#define USE_ACCGYRO_BMI270
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_FLASH
+#define USE_FLASH_M25P16
+#define USE_MAX7456
+#define USE_SERVOS
```
Comment (haslinghuis) [inline configs/default/FLMO-BANDERAF7.config:15]: USE_SERVOS is a build option.
```suggestion

```

File: configs/default/FLMO-BANDERAF7.config [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+set flash_spi_bus = 3
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW270
+set gyro_1_align_pitch = 1800
+set pinio_config = 1,1,1,1
+set pinio_box = 40,41,42,255
+set small_angle = 180
+set craft_name = BANDERA
+set pilot_name = STEPAN
```
Comment (haslinghuis) [inline configs/default/FLMO-BANDERAF7.config:137]: No NAZI's allowed

```suggestion
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/FLMO-BANDERAF7.config [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+dma pin B10 0
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin B11 0
+# pin B11: DMA1 Stream 7 Channel 3
+
+# feature
+feature OSD
+feature RX_SERIAL
+feature TELEMETRY
+feature GPS
```
Comment (haslinghuis) [inline configs/default/FLMO-BANDERAF7.config:97]:
```suggestion

```

File: configs/default/FLMO-BANDERAF7.config [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+serial 1 1 115200 57600 0 115200
+serial 2 2048 115200 57600 0 115200
+serial 4 2 115200 57600 0 115200
+
+# beacon
+beacon RX_LOST
+beacon RX_SET
+
+# master
+set serialrx_provider = CRSF
```
Comment (haslinghuis) [inline configs/default/FLMO-BANDERAF7.config:110]: Cloud based

```suggestion

```

**Comment by haslinghuis** [comment]
@cvetaevvitaliy We are not going to allow the BANDERAF7 target naming. We are kind of shocked using ideology in FOSS. Can't wait for HITLERH7 or TRUMANH7 :grinning: 

Please leave politics = military outside of our project. Betaflight is for civilian use only. Any violations will be punished :rofl: 

As this PR is based upon master we cannot allow merging as we need a feature branch.

Note: bandera in Spanish means flag.

**Comment by haslinghuis** [comment]
@cvetaevvitaliy is the AI companion board sample ready. We like to investigate added functionality for civilian purposes.

Not build sky-net robots :grin: 

Life is about creation - not destruction.

**Comment by cvetaevvitaliy** [comment]
> @cvetaevvitaliy is the AI companion board sample ready. We like to investigate added functionality for civilian purposes.
> 
> 
> 
> Not build sky-net robots :grin: 
> 
> 
> 
> Life is about creation - not destruction.

Tell that to the orcs who are killing my people on my land.

Github is not the place to discuss such topics.

**Comment by cvetaevvitaliy** [comment]
Yep! This is the "FLAG" F7

I will fix the branch from the feature-branch, thank you understanding.

**Pull request closed without merging**